pages/common/dep.md

```diff
@@ -0,0 +1,23 @@
+# dep
+
+> Tool for dependency management in the Go ecosystem.
+
+- Initialize a directory to be a project:
+
+`dep init`
+
+- Install the project dependencies:
+
+`dep ensure`
+
+- Report the status of the project's depedencies:
+
+`dep status`
+
+- Add a new depedency:
+
+`dep ensure -add {{project_url}}`
+
+- Update the locked versions of all dependencies:
+
+`dep ensure -update`
```

Comment:
Is there a way to manually lock/unlock dependencies? If so, I think it should be mentioned before this example, to introduce the concept.

Reply:
Nope. Unfortunately I don't think there is a way to lock & unlock dependencies. To change the locked version I guess you can set the version `Gopkg.toml` and then run this update command.

How should I communicate the fact that the user is supposed to edit the Gopkg.toml? From what I've seen, most of the descriptions are of a single line, describing what the command should do.

Reply:
This is fine @ravernkoh. @waldyrious - the users are expected to go through the `dep` command. So essentially, until you do an `update`, everybody gets the same version of the dependencies.